src/config.ts

```diff
@@ -51,6 +53,8 @@ const defaultConfig: Config = {
       viewport: null,
     },
   },
+  requestPatternAllowlist: [],
```

Comment:
`network: { allowOrigins: undefined, blockOrigins: undefined }`

keep `undefined` here to allow for merging. Note that if you nest you need to merge nested.